src/main/kotlin/io/vertx/gradle/VertxPlugin.kt

```diff
@@ -141,6 +145,27 @@ class VertxPlugin : Plugin<Project> {
     logger.debug("The shadow plugin has been configured")
   }
 
+  private fun configureJibPlugin(project: Project) {
+    val jibExtension = project.extensions.getByName("jib") as JibExtension
+    val vertxExtension = project.vertxExtension()
+    val tag = if (project.version != Project.DEFAULT_VERSION) project.version else "latest"
+    jibExtension.apply {
+      from { imageConfiguration ->
+        imageConfiguration.image = "openjdk:8-jdk-alpine"
```

Comment:
```suggestion
        imageConfiguration.image = "openjdk:8-jre-alpine"
```
I think 'jre' is what we need